source/api/commands/get.md

```diff
@@ -58,6 +58,8 @@ Option | Default | Description
 
 {% yields sets_dom_subject cy.get %}
 
+Keep in mind, that cy.get() **always** returns an Array of DOM elements. (Same as JQuery does.) All chained methods work on all returned elements.
```

Comment:
`cy.get()`, as well as many other commands, actually return a jQuery Object, not an array. https://learn.jquery.com/using-jquery-core/jquery-object/